source/ranges.tex

```diff
@@ -522,10 +522,10 @@
   \end{itemize}
 
 \item
-  Otherwise, \tcode{(ranges::end(E) - ranges::begin(E))}
+  Otherwise, \tcode{\placeholder{make-unsigned-like}(ranges::end(E) - ranges::begin(E))}\iref{range.subrange}
```

Comment:
Should this be \placeholdernc instead?

Reply:
Should it?  I've seen both used within tcode contexts throughout the spec; it's not clear when to use which - can you/someone clarify?

Reply:
From what I've seen, one's used within `\tcode`s and the other within code blocks.

Reply:
Use the nc form to turn off italic correction in places where it looks bad. I think we have some examples in the style guide.

Reply:
Looks like we don't have anything in the style guide for this. We should :)

Reply:
So what should be used in this case?  And yes, an example in the style guide would be very helpful.

Reply:
My personal rule-of-thumb is to use \placeholdernc if there's something immediately afterwards (no space), and use \placeholder if a space follows.

Reply:
ok, done.

Reply:
> From what I've seen, one's used within `\tcode`s and the other within code blocks.

I was confused. The difference I actually noticed was that one had to be wrapped in `@`s and the other not when in `\tcode`s or not, not whether it had to be `\placeholdernc` or the other.